packages/core/src/di/injectable.ts

```diff
@@ -70,8 +70,8 @@ export interface Injectable {
    * - 'root' : The application-level injector in most apps.
    * - 'platform' : A special singleton platform injector shared by all
    * applications on the page.
-   * - 'any' : Provides a unique instance in every module (including lazy modules) that injects the
-   * token.
+   * - 'any' : Provides a unique instance in each lazy loaded module while all eagerly loaded
+   * modules will share an unique instance.
```

Comment:
I must have missed this before. Pardon me. Can we try this?
```suggestion
   * modules share one instance.
```

Reply:
Done.